backend/controller/users.py

```diff
@@ -51,6 +58,19 @@ def build_newuser_map_dict(self, row):
 #####################################################################################################
 #                                       CRUD OPERATIONS                                             #
 #####################################################################################################
+    def userLogin(self, json):
+        email = json["email"]
+        password = json["password"]
+
+        dao = UsersDAO()
+        user = dao.userLogin(email, password)
+        dao.conn.close()
+
+        if user:
+            result = self.build_login_map_dict(user)
+            return jsonify(result), 200
+        else:
+            return jsonify("USER AUTHENTICATION FAILED"), 500
```

Comment:
Code 401 would be more appropiate than 500. 500 is used for a server error, 401 is for Unauthorized credentials.

Reply:
done.